docs/sources/configure-server/reference-configuration-parameters/index.md

```diff
@@ -543,6 +543,183 @@ pool_config:
   # Timeout for ingester client healthcheck RPCs.
   # CLI flag: -distributor.health-check-timeout
   [remote_timeout: <duration> | default = 5s]
+
+ring:
+  # The key-value store used to share the hash ring across multiple instances.
+  kvstore:
+    # Backend storage to use for the ring. Supported values are: consul, etcd,
+    # inmemory, memberlist, multi.
+    # CLI flag: -distributor.ring.store
+    [store: <string> | default = "memberlist"]
```

Comment:
For some reason, the distributor's ring configuration was omitted from the documentation, even though it is still available in the CLI help message. I've added it to the config spec doc.